### ISSUE TITLE
Wet voice

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10572]
**Preferred term label:**

**Synonyms**
gurgly voice

**Definition (free text, please give PubMed ID)**
PMID: 20301305 Oculopharyngeal muscular dystrophy (OPMD) is characterized by ptosis and dysphagia due to selective involvement of the muscles of the eyelids and pharynx, respectively. Early manifestations of dysphagia include increased time needed to consume a meal and an acquired avoidance of dry foods. The severity of dysphagia is the major determinant of prognosis, as it leads to potentially life-threatening aspiration pneumonia and poor nutrition. Other manifestations, observed as the disease progresses, are limitation of upward gaze, tongue atrophy and weakness, chewing difficulties, **wet voice**, facial muscle weakness, axial muscle weakness, proximal limb girdle weakness predominantly in lower limbs, and proximal upper extremity weakness.

Wet voice is a perceptual vocal quality that is commonly used as an indicator of penetration and/or aspiration in clinical swallowing assessments and bedside screening tests. Our aim was to describe the clinimetric characteristics of this clinical sign using various fluid materials and one solid food in the Parkinson's disease (PD) population. 

**Parent term (use hpo.jax.org/app)**
Abnormality of the voice HP:0001608

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:270 Oculopharyngeal muscular dystrophy
ICD11: 8A00.0 Parkinson disease

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Adam Vogel:

 Wet voice is a perceptual vocal quality that is gurgly or liquid sounding and is due to the voice being produced through moisture. It is commonly used as an indicator of penetration and/or aspiration in clinical swallowing assessments and bedside screening tests

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001011
- label: Wet voice
- parents: [Abnormality of the voice (HP:0001608)](https://hpo.jax.org/app/browse/term/HP:0001608)
- pmid: PMID:25001523
- def: Wet voice is a perceptual vocal quality that is gurgly or liquid sounding and is due to the voice being produced through moisture. It is commonly used as an indicator of penetration and/or aspiration in clinical swallowing assessments and bedside screening tests.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10572
- synonyms:
